byzer-lang/en-us/introduction/byzer_lang_design.md

```diff
@@ -0,0 +1,80 @@
+# Byzer Language Design Principle
+
+### 设计理念
```

Comment:
still in progress？

Reply:
now can be merged, we will do the 1st review in the hosted environment.